python-spec/src/somacore/experiment.py

```diff
@@ -65,8 +63,14 @@ class Experiment(
     ms = _mixin.item[_MeasColl]()
     """A collection of named measurements."""
 
-    spatial = _mixin.item[_SceneColl]()
-    """TODO Add docs for spatial collecion."""
+    spatial = _mixin.item[_SceneColl]()  # TODO: Discuss the name of this element.
+    """A collection of named spatial scenes."""
+
+    obs_scene = _mixin.item[_DF]()  # TODO: Rename this element.
```

Comment:
Should we also have a join table between var/scene?

Reply:
@jp-dark - 

1. Am I correct in understanding that this is a dataframe that captures the _many-to-many_ relationship between observations and scenes?

That is, this is a dataframe that maps the primary key in `obs` (`obs_id`) to the primary key for a scene which is of type `string` that can be used to index the `spatial` collection in `Experiment`?

2. Will this dataframe store anything more than just a mapping of primary keys of `obs` to primary keys of `scenes`?

Reply:
@jp-dark - I am also inclined to agree with using a `string` for `scene_id` and removing the indirection. We can always add it later if we think it is useful but right now it seem more straightforward given that Collections are indexed by a string key and if metadata about a scene needs to be looked up in a dataframe, it seems simpler to use that same string key as the primary key in such a "metadata dataframe"

Reply:
I'll defe to you guys, but in our last conversation we had settled on integers. Maybe we should revisit the conversation?

> Should we also have a join table between var/scene?

I think so, it may not be very useful for Visium/Slide-seq but likely useful for Xenium

Reply:
After discussion we decided to use strings for `scene_id` for the time being.